modules/verification-tools.php

```diff
@@ -22,7 +22,7 @@ function jetpack_verification_tools_loaded() {
 add_action( 'jetpack_modules_loaded', 'jetpack_verification_tools_loaded' );
 
 function jetpack_verification_tools_configuration_url() {
-	return admin_url( 'tools.php' );
+	return admin_url( 'admin.php?page=jetpack#/settings?term=site%20verification' );
```

Comment:
Since you are now redirecting to the default location, you could remove the whole `jetpack_verification_tools_configuration_url` function and only rely on `Jetpack::enable_module_configurable( __FILE__ );`, just like you do for WordAds below.

Reply:
Yup, total oversight on my part. Fixed in 0cf3b6c.